docs/overview.md

```diff
@@ -0,0 +1,98 @@
+
+Learn more about [Octopus Cloud](/docs/octopus-cloud/index.md).
+
+### The Octopus Web Portal
+
+Whether you're self-hosting the Octopus Server, or using Octopus Cloud, the Octopus Web Portal is where you'll manage your infrastructure, projects, access the built-in repository, grant your team access to projects, and create your automated deployments.
+
+## Infrastructure
+
+Octopus Deploy organizes your deployment targets (the machines and services you deploy software to) into groups called environments. Typical environments are **Development**, **Test**, and **Production**.
+
+With Octopus Deploy your deployment targets could be Windows servers, Linux servers, Microsoft Azure, AWS, Kubernetes Clusters, Cloud Regions, or even an Offline Package Drop.
+
+Organizing your infrastructure into environments lets you define your deployment processes (no matter how many steps or deployment targets are involved) and have Octopus deploy the right versions of your software, with the right configuration, to the right environments at the right time.
+
+Learn more about managing your [infrastructure](/docs/infrastructure/index.md).
+
+## Packaging Applications
+
+Before you can deploy software with Octopus Deploy, you need to bundle all the files required for the software to run into a supported package. The package must be versioned and stored in a repository. Octopus Deploy includes a built-in repository, but you can also host your packages in external repositories. We recommend configuring your existing tool chain to push packages automatically to the repository. Octopus supports a variety of package formats.
+
+Learn more about [packaging your applications](/docs/packaging-applications/index.md) or how to automate your existing tool chain to push packages to your Octopus Deploy server with our [Build Server Integrations](/docs/octopus-rest-api/index.md).
+
+## The Deployment Process
+
+Octopus Deploy is designed to work with teams following agile software development methodologies, that is, continuously deploying software, iterating, making changes, and redeploying.
+
+Before you can deploy your software, you need to create a project which will include your deployment process and all the information needed for your teams to successfully redeploy every time. The deployment process is a series of steps, each step contains a specific action (or set of actions) that is executed as part of the deployment process every time your software is deployed. Octopus Deploy includes a range of built in steps you can use in your deployment processes.
+
+As part of the deployment process, you can define variables that change based on the scope of your deployments, this means, for instance, you can use different connection strings when you deploy to different environments.
+
+After the initial setup, your deployment process shouldn't change between deployments even though the software being deployed will change as part of the development process.
```

Comment:
I'm not sure about this one. A lot of projects are static, but some have new steps / components added over time. Or are we talking about the process being snapshotted for the release and it won't change for _that release_ during deployments?

Reply:
That's a really good point. I want to keep this page fairly simple (so will leave snapshotting for the deployment process docs). I've updated that paragraph to read:
After the initial setup, you can deploy your software as often as makes sense for you and your customers. You can even update the deployment process over time as the needs of the project change.